core.html

```diff
@@ -81,12 +84,26 @@ <h2>Method Specific Identifier</h2>
 <section class="normative">
     <h2>Generation Method</h2>
 
-    <p>The unique <em>numeric basis</em> underlying a Peer DID MUST be generated as follows:</p>
+    <p>The unique <em>numeric basis</em> underlying a Peer DID MUST be generated in one of the following two ways:</p>
+
+    <h3>Method 0: inception key without doc</h3>
+    <p>If <code>numalgo</code> == <code>0</code>, a single keypair is chosen, having all possible privileges with respect
+        to the DID. The public key value for this pair is the numeric basis of the DID. The genesis version of the DID
+        doc is considered to be empty, granting all privileges to this key (and algorithmic derivations thereof, to
+        provide key agreement versus signing and so forth). The DID doc offers no endpoint. This makes the DID
+        functionally equivalent to a did:key value, and visually similar, except that a peer DID will have the numeric
+        algorithm as a prefix, before the multibase encoded, multicodec-encoded public key. For example, <code>did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH</code> is equivalent to <code>did:peer:<span style="color:blue">0</span>z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH</code>. The did:peer version differs
+        in that it is upgradeable to a dynamically updatable DID with full DID doc and endpoints, simply using deltas,
+        as long as the first delta is authenticated by the inception key.
```

Comment:
Given the discussion in DIF DIDComm about zero trip (or more realistically, minimized trip) connection establishment, I think this reduces the effectiveness of `did:peer` because there are at least two messages from each side to establish a useful DID - this plus the the delta that adds an endpoint service.  I think this makes this version unlikely to be ever used.

Am I missing something on how this would be practically deployed?

Reply:
The delta that adds the endpoint service does not need to be distinct from some other delta. A delta can contain any number of changes, as long as all of them are authorized by the same key(s).